index.html

```diff
@@ -2209,6 +2211,80 @@ <h2>Definition of Roles</h2>
 				</tbody>
 			</table>
 		</div>
+    <div class="role" id="deletion">
+      <rdef>deletion</rdef>
+      <div class="role-description">
+        <p>A deletion contains content that is marked as removed or content that is being suggested for removal. See related <rref>insertion</rref>.</p>
+        <p>Deletions are typically used to either mark differences between two versions of content or to designate content suggested for removal in scenarios where multiple people are revising content.</p>
+      </div>
+      <table class="role-features">
+        <caption>Characteristics:</caption>
+        <thead>
+          <tr>
+            <th scope="col">Characteristic</th>
+            <th scope="col">Value</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <th class="role-abstract-head" scope="row">Is Abstract:</th>
+            <td class="role-abstract"> </td>
+          </tr>
+          <tr>
+            <th class="role-parent-head" scope="row">Superclass Role:</th>
+            <td class="role-parent"><rref>section</rref></td>
+          </tr>
+          <tr>
+            <th class="role-base-head" scope="row">Base Concept:</th>
+            <td class="role-base"> </td>
+          </tr>
+          <tr>
+            <th class="role-related-head" scope="row">Related Concepts:</th>
+            <td class="role-related"><a href="https://www.w3.org/TR/html5/edits.html#the-del-element"><abbr title="Hypertext Markup Language">HTML</abbr> <code>del</code></a></td>
+          </tr>
+          <tr>
+            <th class="role-scope-head" scope="row">Required Context Role:</th>
+            <td class="role-scope"> </td>
+          </tr>
+          <tr>
+            <th class="role-mustcontain-head" scope="row">Required Owned Elements:</th>
+            <td class="role-mustcontain"> </td>
+          </tr>
+          <tr>
+            <th class="role-required-properties-head">Required States and Properties:</th>
+            <td class="role-required-properties"> </td>
+          </tr>
+          <tr>
+            <th class="role-properties-head" scope="row">Supported States and Properties:</th>
+            <td class="role-properties"> </td>
+          </tr>
+          <tr>
+            <th class="role-inherited-head" scope="row">Inherited States and Properties:</th>
+            <td class="role-inherited">Placeholder</td>
```

Comment:
I was confused by the "Placeholder" value here for a second (given that there are HTML/ARIA concepts of a placeholder), maybe we should note "TBD, pending final Superclass Role" to make it clearer to the casual reader and our future selves? Not something I'd block merge on, though.